config.inc.php

```diff
@@ -562,6 +570,18 @@ function x_struct_admin_modify($struct) {
 // http://www.php.net/manual/en/function.imap-open.php
 $CONF['create_mailbox_subdirs_hostoptions'] = array('');
 
+// Optional:
+// Allows a user to reset his forgotten password with a code sent by email/SMS
+$CONF['forgotten_user_password_reset'] = true;
+// Allows an admin to reset his forgotten password with a code sent by email/SMS
+$CONF['forgotten_admin_password_reset'] = true;
+
+// Clickatell gateway to send SMS code for password reset
+// API type: HTTP
+$CONF['clickatell_api_id'] = '';
+$CONF['clickatell_user'] = '';
+$CONF['clickatell_password'] = '';
+$CONF['clickatell_sender'] = '';
```

Comment:
I'd guess there are other services for sending SMS, so it might be a good idea to change this to `$CONF['sms_send_function'] = '';` and then provide the sendCodebySMS() function as an example directly in config.inc.php. Maybe even commented out - the api ID etc. need to be filled in anyway, so removing the comment signs wouldn't be hard.